include/aspect/simulator.h

```diff
@@ -806,8 +807,8 @@ namespace aspect
        * This function is implemented in
        * <code>source/simulator/solver_schemes.cc</code>.
        */
-      void assemble_and_solve_defect_correction_Stokes(DefectCorrectionResiduals &dcr,
-                                                       const bool use_picard);
+      void do_one_defect_correction_Stokes_step(DefectCorrectionResiduals &dcr,
+                                                const bool use_picard);
```

Comment:
I am a bit hesitant to merge this, because all of the similar functions follow the pattern of `assemble_and_solve_...` for all equation systems. How do you feel about `assemble_and_solve_one_defect_correction_Stokes_step`?

Reply:
My reasoning was that these functions appear similar, but are not in fact. The `assemble_and_solve_stokes()` function, for example, has 60 lines and really only does that: It assembles and solves the Stokes problem. But the function here has 180 lines, and it does a lot more: It sets up assemblers, it figures out a lot of tolerances and parameters for the nonlinear scheme, if the residual is already small it doesn't actually solve anything at all, etc. It might be useful to break an `assemble_and_solve_...()` function out of this one, but I don't think it actually fits the scheme of the other functions.